minitwit-backend/minitwit-backend.API/Data/UserRepository.cs

```diff
@@ -147,6 +147,12 @@ public async Task UnFollow(int fromId, int toId)
             await _context.SaveChangesAsync();
         }
 
+        public async Task<APISimFollowing> GetFollowering(int userId)
+        { 
+            var following = await _context.Users.Include(x => x.Following).Where(u => u.UserId.Equals(userId)).SelectMany(u => u.Following).Select(u => u.Username).ToListAsync();
```

Comment:
Wouldn't it make more sense to get the user from the userid and just doing ``return user.Followers``

Reply:
yEs mAyBe gOoD idEa